frontend-react/.storybook/UtilizedUSWDSComponents/SideNav.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import { SideNav } from "@trussworks/react-uswds";
+import React from "react";
+
+export default {
+    title: "Components/Side navigation",
```

Comment:
Should this be Components/SideNav

Reply:
Since all the other component titles are full words, I think it makes sense to leave it as is so it appears as `Side navigation` in the Storybook component sidebar.